Lib/test/test_import/__init__.py

```diff
@@ -877,6 +872,14 @@ def test_pyc_always_writable(self):
             m = __import__(name)
             self.assertEqual(m.x, 'rewritten')
 
+    def test_double_const(self):
```

Comment:
Why do you move the method in FilePermissionTests: "tests for file mode on cached .pyc files"?

Reply:
I don't remember :)
I will move it back.